src/app/results/page.tsx

```diff
@@ -29,15 +29,18 @@ const ResultsContent = () => {
   const [coursesToDisplay, setCoursesToDisplay] = useState<Course[]>([]);
   const [loading, setLoading] = useState(true);
   const [selectedProfessor, setSelectedProfessor] = useState<string | null>(null);
-  const [selectedYear, setSelectedYear] = useState<string | null>(null);
```

Comment:
We don't want to hardcode Spring 2024, we can not assume that the most recent semester that a course was offered is Spring 2024. All changes related this should be removed.